app/Http/Requests/UserRequest.php

```diff
@@ -25,31 +26,11 @@ public function authorize()
     public function rules()
     {
         return [
-            'name' => [
-                'required',
-                'min:2',
-                'not_regex:/^[@#$%&*]/',
-                new ValidateUserName(),
-            ],
-            'email' => 'required|email|not_regex:/^[root]/',
-            'password' => 'required|min:8|regex:/(?=.*?[@#$%&*])/|confirmed',
-            'facebook' => 'required|url',
-            'youtube' => 'required|url'
+            'name' => ['required',  'min:2','not_regex:/^[@#$%&*]/',new ValidateUserName(),],
+            'email' => ['required','email',Rule::unique('users')->ignore($this->user), ],
```

Comment:
'email' => ['required','email',Rule::unique('users')->ignore($this->user), ],
=>
 'email' => ['required','email',Rule::unique('users')->ignore($this->user)],